blivet/devicetree.py

```diff
@@ -1552,7 +1552,14 @@ def addLV(lv):
                         lv_device.cache = LVMCache(lv_dev, cache_size, exists=True)
                 self._addDevice(lv_device)
                 if flags.installer_mode:
-                    lv_device.setup()
+                    try:
+                        lv_device.setup()
+                    except LVMError:
+                        # The only scenario in which this is expected is an incomplete VG
+                        # and/or multiple VGs with the same name.
+                        log.warning("failed to activate lv %s -- ignoring it", lv_device.name)
+                        self._removeDevice(lv_device)
```

Comment:
Do we really want to pretend the LV doesn't exist if we fail to activate it? There are more scenarios coming to my mind in which this may fail - like missing tools for thin metadata consistency checks, etc.